public/styles/styles-login.css

```diff
@@ -0,0 +1,19 @@
+body {
+    background-color:rgb(234, 234, 234);
```

Comment:
@lsimpson667 You can use the CSS variables that is listed in the style guide to get the color you need. Please make this change to all values that is a color.